src/transfer.jl

```diff
@@ -153,3 +121,49 @@ function transfer_blochwave(ψ_in, basis_in::PlaneWaveBasis{T},
         transfer_blochwave_kpt(ψ_in[ik], basis_in, basis_in.kpoints[ik], basis_out, kpt_out)
     end
 end
+
+"""
+Find the equivalent index of the coordinate `kcoord` ∈ ℝ³ in a list `kcoords` ∈ [-½, ½)³.
+`ΔG` is the vector of ℤ³ such that `kcoords[index] = kcoord + ΔG`.
+"""
+function find_equivalent_kpt(basis::PlaneWaveBasis{T}, kcoord, spin; tol=sqrt(eps(T))) where {T}
```

Comment:
Coverage says this function is untested. Can you check you did not mess up the tests accidentially.